.github/workflows/features.yml

```diff
@@ -15,9 +15,7 @@ name: feature tests
 # trigger workflows automatically but will wait for approval from somebody
 # with write access.
 # See: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks
-on:
-  pull_request:
-    branches: [main]
+on: [pull_request]
```

Comment:
I don't know why I set this before but it is also nice to run CI on branches which have a different base than `main`.
Example: This change set!

Do you see a problem with this?

Reply:
Sounds great!